components/webpack/webpack.config.js

```diff
@@ -25,7 +26,7 @@ module.exports = async function (env, argv) {
       alias: pathMap,
       // For explanation of "chromeapp", see:
       // https://github.com/brave/brave-browser/issues/5587
-      aliasFields: ['chromeapp', 'browser']
+      aliasFields: isTrezorEntrypoint ? ['browser'] : ['chromeapp']
```

Comment:
I think this would be better with an alias-fields cli param / env variable. Every other customization is passed in this way so I think that would be consistent.

We would add the GN param handling to typescript.gni which would pass the extra cli param to the Python script, which would pass the env variable / param to the webpack NodeJS script.

Reply:
done